.github/workflows/docs.yml

```diff
@@ -11,6 +11,7 @@ concurrency:
 
 env:
   BAYBE_DOCS_LINKCHECK_IGNORE: true  # Ignore link checks to github.io (pages might not yet exist)
+  SMOKE_TEST: true 
```

Comment:
@Scienfitz : Since I pushed that after your first round of comments, I just wanted to make you aware of this addition. With you being the pipeline expert, I'd highly appreciate if you could verify that this is properly included here :)

Reply:
why put it here? shouldnt it go into tox.ini? It also must be done for all coretests and fulltests, right?

Reply:
True, we should just have it set to True in all of these tests. We then do not need it in the workflows at all right?

Reply:
yes if you put it into docs and test environments of tox via `setenv` then i can run it fast locally as well as on github actions. if i want the full example run I need to call it via `python example.py`
would make sense to me

Reply:
@AVHopp I pushed a commit shifting the entire `SMOKE_SCREEN` responsibility to the tox environments

all pipelines and local calls to tox will now by default run `fulltest`, `coretest` and `docs` in `SMOKE_SCREEN`

Any call to pytest or python run_example will run the full example

Reply:
Thank you, I'll check that out soon and then resolve this comment.